### ISSUE TITLE
Prevent vertical scroll in quotes / code blocks / `pre`

### ISSUE THREAD

**Issue by ArturGajowy** [zulip/zulip-desktop#788]
Hi!

Pasting a long line of text in a quote causes a vertical scrollbar to appear inside a message, which prevents the quote from being readable. E.g., paste the following docs excerpt into a quote block in a message:

```
Linkifiers make it easy to refer to issues or tickets in third party issue trackers, like GitHub, Salesforce, Zendesk, and others. For instance, you can add a filter that automatically turns #2468 into a link to  https://github.com/zulip/zulip/issues/2468.

If the pattern appears in a message topic, Zulip adds a little button to the right of the topic that links to the appropriate URL.
```

(funny how github UI suffers from exactly the same issue :smile:)

Currently, I'm adding the following user css to prevent vertical scrollbar in a pre block:

```css
.messagebox-content pre {
    white-space: pre-wrap;
    word-break: break-word;
}
```

Could we have sth similar added by default? I can do a PR if you tell me where the message styles are in the source code :)

Thanks!

---

- **Operating System**:
  - [X] Windows
  - [X] Linux/Ubuntu
  - [X] macOS
- **Clear steps to reproduce the issue**:
- **Relevant error messages and/or screenshots**:

**Comment by kanishk98** [comment]
Hello @ArturGajowy! Thanks for filing the issue. I think this is related to our web app and not the desktop app, so I'll need to ask @timabbott to address your questions here.

**Comment by ArturGajowy** [comment]
oh, so you're saying I can contribute to to Zulip's look & feel by contributing there? :)

**Comment by kanishk98** [comment]
Yes, we'd love to see your contributions. The front-end code for the web app and the server-side Python stuff all lives in github.com/zulip/zulip.
